packages/fixie/package.json

```diff
@@ -1,56 +1,32 @@
 {
-  "name": "fixie",
-  "version": "1.0.0",
-  "devDependencies": {
-    "@tsconfig/node18": "^2.0.1",
-    "@types/lodash": "^4.14.195",
-    "@types/node": "^20.3.1",
-    "@types/prompt-sync": "^4.2.0",
-    "@types/uuid": "^9.0.2",
-    "@types/yargs": "^17.0.24",
-    "@typescript-eslint/eslint-plugin": "^5.60.0",
-    "@typescript-eslint/parser": "^5.60.0",
-    "eslint": "^8.40.0",
-    "eslint-config-nth": "^2.0.1",
-    "typescript": "^5.1.3"
-  },
-  "private": true,
-  "type": "module",
-  "scripts": {
-    "dev": "yarn build && npx ts-node dist/serve-bin.js --port 8080 --package-path fixie",
-    "lint": "eslint . --max-warnings 0",
-    "lint:fix": "eslint . --fix",
-    "typecheck": "tsc -p tsconfig.json",
-    "build": "yarn run typecheck",
-    "bin:dev": "npx ts-node dist/serve-bin.js --port 8080 --package-path fixie"
-  },
-  "dependencies": {
-    "@opentelemetry/api": "^1.4.1",
-    "@opentelemetry/api-logs": "^0.41.1",
-    "@opentelemetry/exporter-logs-otlp-grpc": "^0.41.1",
-    "@opentelemetry/exporter-trace-otlp-grpc": "^0.41.1",
-    "@opentelemetry/instrumentation-fastify": "^0.32.0",
-    "@opentelemetry/instrumentation-fetch": "^0.41.1",
-    "@opentelemetry/instrumentation-http": "^0.41.2",
-    "@opentelemetry/sdk-logs": "^0.41.1",
-    "@opentelemetry/sdk-node": "^0.41.1",
-    "ai-jsx": "*",
-    "dotenv": "^16.3.1",
-    "fastify": "^4.20.0",
-    "yargs": "^17.7.2"
-  },
-  "bin": {
-    "serve-bin": "dist/serve-bin.js"
-  },
-  "exports": {
-    ".": {
-      "import": {
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.js"
-      },
-      "require": {
-        "default": "./dist/index.cjs"
-      }
-    }
-  }
+	"name": "fixie",
+	"version": "1.0.0",
+	"private": true,
```

Comment:
How are we thinking of distributing this binary? Is it a standalone npm package? Or is it bundled with ai-jsx?

If it's the former, we need to remove `private: true`.
If it's the latter, then we shouldn't make a new monorepo package for the CLI, but should put it instead in `packages/ai-jsx`.

I'm not totally sure which we want. I'm curious to chat through it with @psalas.

Reply:
If we're moving the whole CLI SDK to node, then a standalone `fixie` package seems right to me.